drivers/serial/serial.c

```diff
@@ -1408,9 +1469,9 @@ static ssize_t uart_write(FAR struct file *filep, FAR const char *buffer,
    */
 
   uart_disabletxint(dev);
-  for (; buflen; buflen--)
+  for (; buflen; uio_advance(uio, 1), buflen--)
```

Comment:
merge line 1474

Reply:
sorry, i'm not sure which line you were referring to.